src/main/java/seedu/address/logic/parser/RecipeBookSyntax.java

```diff
@@ -0,0 +1,14 @@
+package seedu.address.logic.parser;
+
+/**
+ * Contains Command Line Interface (CLI) syntax definitions common to multiple commands.
+ */
+public class RecipeBookSyntax {
+
+    /* Prefix definitions */
+    public static final Prefix PREFIX_NAME = new Prefix("-n");
```

Comment:
Similar to Aidan's comment, might be difficult for users to remember which prefix requires longhand and shorthand prefixes. Maybe we can discuss more about this.